runtime/queries/sshclientconfig/highlights.scm

```diff
@@ -1,17 +1,17 @@
-(host) @keyword
-(host_value) @identifier
+(host) @namespace
+(host_value) @string
 
-(match) @keyword
-(match_value) @identifier
+(match) @namespace
+(match_value) @string
 
 (add_keys_to_agent) @keyword
-(add_keys_to_agent_value) @boolean
+(add_keys_to_agent_value) @constant.builtin.boolean
 
 (address_family) @keyword
-(address_family_value) @type
+(address_family_value) @constant.builtin.string
```

Comment:
`constant.builtin.string` is not defined: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

this could be `constant.builtin` or `string`

Reply:
ah great! I was looking for that doc but somehow missed it. I've adjusted all `constant.builtin.string` usages to either be `constant.builtin` for those config options that have a predefined set of acceptable values and `string` for everything else.